app/models/user.rb

```diff
@@ -21,11 +21,13 @@ class User < ApplicationRecord
 
   has_secure_password
 
-  has_one_attached :avatar
+  unless Rails.env.test?
```

Comment:
Hum... don't be mad at me for bothering you again xD creating different flows for test and production is quite dangerous. this should never happen. solution should be achieved on rspec side, not on code. However, it was a nice hammer time mode from an engineering point of view xD

In theory, when running the avatar upload locally/test, it should store files locally. Or perhaps you can simply mock the avatar uploader, so that when it's used, it basically does nothing. For eg this line     `path = "C:/Galvao/skillX/app/uploaders/avatar_uploader.rb/../../public/uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}" ` is suspicious.. having your path hardcoded is a smell that something isn't properly configured. 

@dpflucas or @laginha or @pdcribeiro  can u help him setting this better? you have way more knowledge than me at this stage x)